src/domain/models/VerifiableCredential.ts

```diff
@@ -1,8 +1,62 @@
+
 export enum CredentialType {
-  JWT = "jwt",
+  JWT = "prism/jwt",
   W3C = "w3c",
   AnonCreds = "AnonCreds",
-  Unknown = "Unknown",
+  ANONCREDS_OFFER = "anoncreds/credential-offer@v1.0",
+  ANONCREDS_REQUEST = "anoncreds/credential-request@v1.0",
+  ANONCREDS_ISSUE = "anoncreds/credential@v1.0",
+  ANONCREDS_PROOF_REQUEST = "anoncreds/proof-request@v1.0",
+  PRESENTATION_EXCHANGE_DEFINITIONS = "dif/presentation-exchange/definitions@v1.0",
+  PRESENTATION_EXCHANGE_SUBMISSION = "dif/presentation-exchange/submission@v1.0",
```

Comment:
I'm confused by this, are these CredentialTypes? look more like Message types?

Reply:
They where mostly shared, except for W3C const.. I have now duplicated them in MessageAttachment and Domain.CredentialType

export enum AttachmentFormats {
  ANONCREDS_OFFER = "anoncreds/credential-offer@v1.0",
  ANONCREDS_REQUEST = "anoncreds/credential-request@v1.0",
  ANONCREDS_ISSUE = "anoncreds/credential@v1.0",
  ANONCREDS_PROOF_REQUEST = "anoncreds/proof-request@v1.0",
  PRESENTATION_EXCHANGE_DEFINITIONS = "dif/presentation-exchange/definitions@v1.0",
  PRESENTATION_EXCHANGE_SUBMISSION = "dif/presentation-exchange/submission@v1.0",
  JWT = "prism/jwt",
  AnonCreds = "AnonCreds",
}

export enum CredentialType {
  JWT = "prism/jwt",
  W3C = "w3c",
  AnonCreds = "AnonCreds",
  Unknown = "Unknown"
}